rllib/agents/a3c/a3c_tf_policy.py

```diff
@@ -129,4 +158,6 @@ def setup_mixins(policy, obs_space, action_space, config):
     postprocess_fn=postprocess_advantages,
     extra_action_fetches_fn=add_value_function_fetch,
     before_loss_init=setup_mixins,
-    mixins=[ValueNetworkMixin, LearningRateSchedule])
+    mixins=[ValueNetworkMixin, LearningRateSchedule],
+    view_requirements_fn=view_requirements_fn_w_vf_preds,
```

Comment:
Can we call this `train_view_requirements` to clear up ambiguity with the inference view?